check_modulemd.py

```diff
@@ -133,8 +133,12 @@ def test_description(self):
             "Checking for presence of description that is properly punctuated")
         if self.mmd.description and len(self.mmd.description) > 0:
             if not self.mmd.description.endswith('.'):
-                self.error("Description should end with a period: %s" %
-                           self.mmd.description)
+                if self.mmd.description.endswith('.\n'):
```

Comment:
what if there are more newlines there? you could compare description.rstrip() vs description